### ISSUE TITLE
Click on file and line in IPython console Traceback doesn't jump to code in Editor

### ISSUE THREAD

**Issue by gerazov** [spyder-ide/spyder#15213]
Congrats on Spyder 5! :partying_face: Awesome job!

Seems that clicking on the Traceback in the IPython console is still there (the mouse changes as for a link), but jumping to the line and file in the Editor doesn't work, i.e. nothing happens.

Minor inconvenience, just wanted to report it.

* Spyder version: 5.0.0 None
* Python version: 3.6.13 64-bit
* Qt version: 5.12.9
* PyQt5 version: 5.12.3
* Operating System: Linux 5.4.105-1-MANJARO

### Dependencies

I've made this into a summary - it keeps the issues tidy. Might be useful to update the template :+1: 

<details>
<summary>Dependency listing</summary>
<p>

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  1.6.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.0.2 (OK)
IPython >=7.6.0               :  7.16.1 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.0.1 (OK)
nbconvert >=4.0               :  6.0.7 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.8.1 (OK)
pylint >=1.0                  :  2.7.2 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=0.5.7             :  1.0.2 (OK)
qtconsole >=5.0.3             :  5.0.3 (OK)
qtpy >=1.5.0                  :  1.9.0 (OK)
rtree >=0.8.3                 :  0.9.7 (OK)
setuptools >=39.0.0           :  49.6.0.post20210108 (OK)
sphinx >=0.6.6                :  3.5.3 (OK)
spyder_kernels >=2.0.1,<2.1.0 :  2.0.1 (NOK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  1.0.2 (OK)
xdg >=0.26                    :  0.26 (OK)
zmq >=17                      :  22.0.3 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=2.0.0            :  None (NOK)
numpy >=1.7                   :  1.19.5 (OK)
pandas >=1.1.1                :  None (NOK)
scipy >=0.17.0                :  1.5.3 (OK)
sympy >=0.7.3                 :  1.4 (OK)
```
</p>
</details>

**Comment by gerazov** [comment]
Duplicate of #15166 closing ...

**Comment by ccordoba12** [comment]
> Congrats on Spyder 5! partying_face Awesome job!

Thanks!

> Duplicate of #15166 closing ...

Yep, thanks for not noticing. We'll fix it in 5.0.1.

**Comment by gerazov** [comment]
The summary is a good thing to add to the issue template - to prevent scrolling through the dependencies ...
